packages/plugin-conversation/src/to-array.js

```diff
@@ -0,0 +1,23 @@
+import {isFunction} from 'lodash';
+
+/**
+ * Helper to convert objects into arrays of transforms. probably belongs in
+ * spark-core
+ * @param {string} direction
```

Comment:
inbound or outbound. we only want to apply encrypt transforms for stuff we're sending and decrypt on stuff we're receiving.